src/plugins/chart_expressions/expression_metric/public/components/metric_vis.tsx

```diff
@@ -133,7 +133,7 @@ const buildFilterEvent = (rowIdx: number, columnIdx: number, table: Datatable) =
 const getIcon =
   (type: string) =>
   ({ width, height, color }: { width: number; height: number; color: string }) =>
-    <EuiIcon type={type} width={width} height={height} fill={color} style={{ width, height }} />;
+    <EuiIcon type={type} width={width} height={height} fill={color} css={{ width, height }} />;
```

Comment:
I would remove the HTML attributes but keep the inline style, since width/height appear to be dynamic
```suggestion
    <EuiIcon type={type} fill={color} style={{ width, height }} />;
```